dashboard/pkg/epinio/dialog/ExportAppDialog.vue

```diff
@@ -3,14 +3,21 @@ import GenericPrompt from '@shell/dialog/GenericPrompt';
 import Banner from '@components/Banner/Banner.vue';
 import Tabbed from '@shell/components/Tabbed/index.vue';
 import Tab from '@shell/components/Tabbed/Tab.vue';
-import { APPLICATION_PARTS } from '../types';
 import JSZip from 'jszip';
 import { downloadFile } from '@shell/utils/download';
+import PercentageBar from '@shell/components/PercentageBar';
+import { APPLICATION_PARTS } from '../types';
+
+const partsWeight = {
+  [APPLICATION_PARTS.VALUES]: 0.15,
+  [APPLICATION_PARTS.CHART]:  0.15,
```

Comment:
these should be slightly different to ensure that when the text changes the bar also does

Reply:
This variable is decoupled from the download mechanism, therefore to the text shown. It only assigns a weight into the progress bar for each of the 3 parts downloaded.
We are assuming that the big one is the image.

> the text changes the bar also does

This behavior is implemented in the `fetchPart` function, so that when a new part is about to be downloaded , the text changes:
```
    async fetchPart(resource, part) {
      this.downloadingPart = part;
```

Reply:
ah, hadn't spotted they're cumulative. i'd seen it once when the text changed to something new but the bar stayed the same. will see if it happens again